total-rewards.json

```diff
@@ -1,14 +1,14 @@
 {
   "rewards": {
-    "notAssigned": 2875,
-    "assigned": 4574,
-    "completed": 65819,
-    "total": 73268
+    "notAssigned": 2850,
+    "assigned": 0,
+    "completed": 12035,
+    "total": 14885
   },
   "tasks": {
-    "notAssigned": 313,
-    "assigned": 284,
-    "completed": 345,
-    "total": 597
+    "notAssigned": 36,
+    "assigned": 32,
```

Comment:
This seems like too many assigned! It's a separate issue!

Reply:
from the current dev branch, those small numbers are from my devpool.

Bit of a nightmare working with that from a forked perspective imo, manageable but tedious

```ts
{
  "rewards": {
    "notAssigned": 2850,
    "assigned": 825,
    "completed": 69593,
    "total": 73268
  },
  "tasks": {
    "notAssigned": 312,
    "assigned": 285,
    "completed": 354,
    "total": 597
  }
}
```

Reply:
Interesting stats. Big numbers are always good for marketing.

Reply:
you know what they say about big numbers

Reply:
Big funding